katdal/test/test_chunkstore_s3.py

```diff
@@ -342,6 +344,12 @@ def test_missing_or_empty_buckets(self):
         # Without put_chunk the bucket is empty
         with assert_raises(StoreUnavailable):
             self.store.get_chunk(f'{BUCKET}-empty/x', slices, dtype)
+        # Check that the standard bucket has not been verified yet
+        bucket_url = urllib.parse.urljoin(self.store._url, BUCKET)
+        assert_true(bucket_url not in self.store._verified_buckets)
```

Comment:
```suggestion
        assert_not_in(bucket_url, self.store._verified_buckets)
```

Reply:
Done, thanks! I did not know about `assert_in`.